store/postgres/tests/relational.rs

```diff
@@ -121,12 +123,22 @@ lazy_static! {
         entity.set("id", "one");
         entity.set("bool", true);
         entity.set("int", std::i32::MAX);
-        entity.set("bigDecimal", (*LARGE_DECIMAL).clone());
+        let decimal = (*LARGE_DECIMAL).clone();
```

Comment:
Let's not make this change now, but in the future if you encounter a test which is constructing an entity, consider re-factoring it to use the new `entity!` macro. Ditto for `object!` in GraphQL.

Reply:
Thanks for the tip, I only used the `object!` macro so far.